#!/usr/bin/env python
PACKAGE = "turtle"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("k_rho",          double_t,   0,  "double_param",     0.025,     -2,    2.0)
gen.add("k_beta",         double_t,   0,  "double_param",   -0.0125,     -2,    2.0)
gen.add("k_alpha",        double_t,   0,  "double_param",      0.03,     -2,    2.0)
gen.add("goal_x",         double_t,   0,  "double_param",         0,  -20.0,   20.0)
gen.add("goal_y",         double_t,   0,  "double_param",         0,  -20.0,   20.0)
gen.add("goal_theta",     double_t,   0,  "double_param",         0, -180.0,  180.0)
gen.add("start",          bool_t,     0,  "bool_param"  ,                     False)

exit(gen.generate(PACKAGE, "robotController", "reconfigure"))